docs/reference/system-requirements.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Aurora System Requirements
+description: System requirements to comofortably use Aurora
+---
+
+Using Aurora on your computer does not require any special sauce, but there are some system requirements you should be aware of. Other than the hard requirement of a 64-bit capable CPU (AMD, Intel) you should have at least the following system specifications on your Aurora Box:
+
+- A recent dual-core or higher multicore processor with at least 2 GHz
```

Comment:
Remember: adding this information here means you are going to support it. If someone with a 3rd gen dual core with 4GB or RAM gets mad at you, they will feel like they're in the right

Reply:
Hmm you are not wrong but what I should I put there instead? @nicknamenamenick suggested to write some basic system requirements but I'm not exactly sure what to target here.

Reply:
You should write out minimal requirements and recommended ones: mininmal ones just so people can know how little aurora can take up, and recommended for actually supported hardware